tools/scalar-schema/src/scalar_schema/core.clj

```diff
@@ -1,36 +1,37 @@
 (ns scalar-schema.core
   (:require [clojure.tools.cli :refer [parse-opts]]
             [scalar-schema.cassandra-schema :as cassandra-schema]
-            [scalar-schema.cosmos-schema :as cosmos-schema])
+            [scalar-schema.cosmos-schema :as cosmos-schema]
+            [scalar-schema.dynamo-schema :as dynamo-schema])
   (:gen-class))
 
 (def cli-options
   [[nil "--cassandra" "Operate for Cassandra"]
    [nil "--cosmos" "Operate for Cosmos DB"]
+   [nil "--dynamo" "Operate for DynamoDB"]
    ["-h" "--host DB_HOST" "Address of Cassandra like IP or URI address of your Cosmos DB account"]
-   ["-u" "--user USERNAME" "Username of the database. This option is ignored when Cosmos DB."]
+   ["-u" "--user USERNAME" "Username of the database. This option is ignored when Cosmos DB and DynamoDB."]
    ["-p" "--password ACCOUNT_PASSWORD" "Password of Cassandra or Cosmos DB account"]
    ["-f" "--schema-file SCHEMA_JSON" "Schema file"]
    ["-r" "--ru RESOURCE_UNIT" "Base RU for each table on Cosmos DB. The RU of the coordinator for Scalar DB transaction is specified by this option. This option is ignored when Cassandra."
```

Comment:
It seems like DynamoDB should be mentioned in the description since `-r` is shared with both.

Reply:
Thank you! Fixed!